l2geth/core/state/statedb.go

```diff
@@ -244,6 +246,7 @@ func (s *StateDB) GetBalance(addr common.Address) *big.Int {
 	if rcfg.UsingOVM {
 		// Get balance from the OVM_ETH contract.
 		// NOTE: We may remove this feature in a future release.
+		statedumper.WriteETH(addr)
```

Comment:
Is this code necessary?
I think we don't have to write in case of `GetBalance`,,,🥲 I don't know well,,,